public/usage-examples/geometry/circle_radius/circle_radius-1-simple-top-level.cs

```diff
@@ -0,0 +1,31 @@
+using SplashKitSDK;
+using static SplashKitSDK.SplashKit;
+
+OpenWindow("Circle Radius", 800, 600);
+ClearScreen();
+
+// Set position for the circle
+double x_position = 400;
+double y_position = 300;
+
+ //Create a circle A at the position (x_position, y_position)
+Circle A = CircleAt(x_position, y_position, 200);
+
+// Find the radius of the circle A
+double radius = CircleRadius(A);
+
+//Draw the Circle
```

Comment:
This comment is different across files. Some say "Draw the circle" some say "Fill the circle"